administrator/language/en-GB/plg_fields_note.ini

```
@@ -0,0 +1,15 @@
+; Joomla! Project
+; (C) 2025 Open Source Matters, Inc. <https://www.joomla.org>
+; License GNU General Public License version 2 or later; see LICENSE.txt
+; Note : All ini files need to be saved as UTF-8
+
+PLG_FIELDS_NOTE="Fields - Note"
+PLG_FIELDS_NOTE_LABEL="Note (%s)"
+PLG_FIELDS_NOTE_PARAMS_CLASS_DESC="The class used on the div of the note"
```

```suggestion
PLG_FIELDS_NOTE_PARAMS_CLASS_DESC="The class used on the div of the note."
```
sentences should end with a period